test/e2e/storage/persistent_volumes-local.go

```diff
@@ -132,8 +134,171 @@ var _ = framework.KubeDescribe("[Volume] PersistentVolumes-local [Feature:LocalP
 			f.TestContainerOutput("pod writes to PV", podSpec, 0, []string{testVol.hostDir})
 		})
 	})
+
+	Context("when two pods request one prebound PVC serially", func() {
+		var (
+			testVol *localTestVolume
+			node    *v1.Node
+		)
+
+		BeforeEach(func() {
+			// Choose the first node
+			node = &config.nodes[0]
+		})
+
+		AfterEach(func() {
+			cleanupLocalVolume(config, testVol)
+			testVol = nil
+		})
+
+		It("should be able to mount volume, write from pod1, and read from pod2", func() {
+			By("Initializing test volume")
+			testVol = setupLocalVolume(config, node)
+
+			By("Creating local PVC and PV")
+			createLocalPVCPV(config, testVol)
+
+			By("Creating a pod to write to the PV")
+			testFilePath := filepath.Join("/mnt/volume1", testFile)
+			writeCmd := fmt.Sprintf("echo %s > %s; cat %s", testVol.hostDir, testFilePath, testFilePath)
+			podSpec1 := createLocalPod(config, testVol, writeCmd)
+			f.TestContainerOutput("pod writes to PV", podSpec1, 0, []string{testVol.hostDir})
+
+			By("Creating a pod to read from the PV")
+			readCmd := fmt.Sprintf("cat /mnt/volume1/%s", testFile)
+			podSpec2 := createLocalPod(config, testVol, readCmd)
+			f.TestContainerOutput("pod reads PV", podSpec2, 0, []string{testVol.hostDir})
+		})
+	})
+
+	Context("when two pods request one prebound PVC one after other", func() {
```

Comment:
I see the serially test above is using `TestContainerOutput`, which runs the passed in command to completion, whereas this test is creating a long running pod, execing into it to run the commands, and then destroying the pod.  

I'm curious to why there is difference in behavior (above is not seeing the unmount bug, but this one is)?  Is it that the methods for checking pod termination state are different?  Theoretically, the unmount bug should exist regardless of if the pod is long running or run-once.